asmcli/lib/util.sh

```diff
@@ -723,27 +723,27 @@ get_cr_channels() {
 
 get_cr_yaml() {
   local CHANNEL; CHANNEL="${1}"
-  local EXPERIMENTAL; EXPERIMENTAL="$(context_get-option "EXPERIMENTAL")"
+  local LEGACY; LEGACY="$(context_get-option "LEGACY")"
   local CR REVISION
   case "${CHANNEL}" in
     regular)
-      if [[ "${EXPERIMENTAL}" -eq 1 ]]; then
+      if [[ "${LEGACY}" -eq 0 ]]; then
```

Comment:
You made the `is_legacy` function so use that here instead of checking the variable explicitly.

Reply:
Done.